R/detect_structure.R

```diff
@@ -29,8 +29,12 @@
 #' detected_structure
 detect_structure <- function(reporting_triangle) {
   n_row_nas <- sum(is.na(rowSums(reporting_triangle)))
-  # Structure is NULL if there are no NAs
+  # Structure is 0 if there are no NAs
   if (n_row_nas == 0) {
+    cli_alert_info(
```

Comment:
As written this implies its never okay for this to be the case whereas it is actually part of the supported data format. I think I would either just have this check where it matters for functionalit y or reword it so that its clearer what this might mean.